documentation/tutorial/03-deploy-supporting-charms.md

```diff
@@ -0,0 +1,52 @@
+# Deploy supporting charms
+
+This part of the tutorial focuses on deploying PostgreSQL and Redis charms, which Superset requires to function.
+
+## Deploy a Database
+Charmed Superset relies on an external [Charmed PostgreSQL](https://charmhub.io/postgresql-k8s) database for storing application metadata such as users, dashboard definitions and logs. Deploy it as follows:
+
+```bash
+juju deploy postgresql-k8s --trust
+
+```
+You can check the deployment was successful by running `juju status`. You should expect an output like this:
+
+```
+Model           Controller           Cloud/Region        Version  SLA          Timestamp
+superset-model  superset-controller  microk8s/localhost  3.5.3    unsupported  10:48:10+01:00
+
+App             Version  Status   Scale  Charm           Channel    Rev  Address  Exposed  Message
+postgresql-k8s           waiting    0/1  postgresql-k8s  14/stable  381           no       installing agent
+
+Unit              Workload  Agent       Address  Ports  Message
+postgresql-k8s/0  waiting   allocating                  installing agent
+
+```
+
+[note]
+The database deployment may take some time, up to 10 minutes, to complete. After that, all Juju components should be in `active` status. 
```

Comment:
nitpick: `up to` indicates to me that 10 minutes is a hard limit. A more hedged wording might be better, or we can put more information about other signs to look for such as the debug log.

I also see that similar usage is in other files, any action taken for this should be taken for them as well.

Reply:
I take your point. The tutorial is supposed to provide a single path to success without tangents or expectation of failure, so I've updated the wording to make them less absolute. I have plans to separately add a `how-to` section for debugging but that is outside the scope of this documentation push.